11ty/definitions/neo-colonialism.md

```diff
@@ -0,0 +1,25 @@
+---
+title: neo-colonialism
+slug: neo-colonialism
+speech: noun
+excerpt: the continued influence of colonial era ideologies and practices in a country or region by former dependencies, generally enforced by its own government and/or citizens as a lingering effect of colonialism
+sub_terms:
+    - text: neo-
+    full_title: neo-colonialism
+    - text: modern
+    full_title: modern colonialism
+defined: true
+reading:
+    - text: neo-colonialim - internet encyclopedia of philosophy
+      href: https://iep.utm.edu/neocolon/
```

Comment:
```suggestion
  - text: neo-colonialim - internet encyclopedia of philosophy
    href: https://iep.utm.edu/neocolon/
```
The tabbing is a bit particular in front matter 🙃 Making these updates will make your repo build.

Reply:
got it - will make a note of this for the future